ldap/georchestra.ldif

```diff
@@ -179,6 +179,14 @@ member: uid=testeditor,ou=users,dc=georchestra,dc=org
 member: uid=testreviewer,ou=users,dc=georchestra,dc=org
 member: uid=testadmin,ou=users,dc=georchestra,dc=org
 
+# ORGADMIN, roles, georchestra.org
+dn: cn=ORGADMIN,ou=roles,dc=georchestra,dc=org
+objectClass: top
+objectClass: groupOfMembers
+cn: ORGADMIN
+ou: 24
+description: This group is automaticly granted to all delegated admins
```

Comment:
This role is automatically granted to all users holding an admin delegation

Reply:
So we need to add `testeditor` and `testuser` because a delegation is defined in database :
https://github.com/georchestra/georchestra/pull/1987/files#diff-d891758ffba6a78a5e0fa75847d79feaR50